server/handlers/tunnel.go

```diff
@@ -44,6 +45,11 @@ var (
 
 // Tunnel forwards the request to the service provider's backend
 func InitTunnel(w http.ResponseWriter, r *http.Request) {
+
+	r.Header.Set("BackendURL", "victus_backend.com")
+	clientPortalData := Ctl.GetClientDataByBackendURL(w, r)
```

Comment:
it should be using value from backend variable below, and i think you don't need to create a new controller, just call the service

Reply:
Okay, I will